packages/idos-sdk-js/src/__tests__/kwil-wrapper.test.ts

```diff
@@ -0,0 +1,79 @@
+import { Utils, WebKwil } from "@kwilteam/kwil-js";
+import { Wallet } from "ethers";
+import { implicitAddressFromPublicKey } from "src/lib";
+import { KwilWrapper } from "src/lib/kwil-wrapper";
+import { beforeEach, describe, expect, test } from "vitest";
+
+const kwilProvider = "kwil-provider";
+const chainId = "chain-id";
+const dbId = "db-id";
+
+class TestKwilClient extends WebKwil {
+  constructor() {
+    super({ kwilProvider, chainId });
+  }
+}
+
+let kwilWrapper: KwilWrapper;
+
+describe("kwil-wrapper", () => {
+  beforeEach(() => {
+    kwilWrapper = new KwilWrapper(new TestKwilClient(), kwilProvider, dbId);
+  });
+
+  test("should create a new instance of KwilWrapper", () => {
+    expect(kwilWrapper.client).toBeInstanceOf(WebKwil);
+    expect(kwilWrapper.kwilProvider).toBe(kwilProvider);
+    expect(kwilWrapper.dbId).toBe(dbId);
+  });
```

Comment:
Meh. Unless you wanna make sure these properties don't go away by mistake, I don't see a bunch of value in having these in.